runtime/runtime.m

```diff
@@ -1323,6 +1323,9 @@ -(void) xamarinSetGCHandle: (uint32_t) gc_handle;
 
 	runtime_initialize = mono_class_get_method_from_name (runtime_class, "Initialize", 1);
 
+	if (runtime_initialize == NULL)
+		xamarin_assertion_message ("Fatal error: failed to load the method '%s.%s.%s'\n.", objcruntime, "Runtime", "Initialize");
```

Comment:
Why are "Runtime" and "Initialize" used as string substitutions instead of just going into the format string?

Reply:
It was a copy-paste from somewhere else. But reminds me that this could probably be simplified to use fewer new strings. I'll have a look at that.